src/Compilers/Core/Portable/MetadataReference/ModuleMetadata.cs

```diff
@@ -23,10 +23,15 @@ public sealed partial class ModuleMetadata : Metadata
 
         private readonly PEModule _module;
 
-        private ModuleMetadata(PEReader peReader)
+        private readonly IDisposable? _owner;
+        private readonly bool _disposeOwner;
```

Comment:
That's not the case (and i can doc).  The IDisposable may be passed into the ModuleMetadata (so it can own it and prevent it from being GC'ed as long as it is alive).  However, it may not be responsible for disposing of it ever.  This is controlled by the PEOption.LeaveOpen flag.  e.g. it's possible for someone to pass in the `IDisposable` and `LeaveOpen` which would mean `_owner` would be non-null while `_disposeOwner` can be false.  Will doc.

Reply:
Docs added.